src/base.py

```diff
@@ -258,21 +258,22 @@ def check_html_pdf_metadata(cls, values):
 
         return values
 
-    @root_validator
```

Comment:
Has the input file changed? How are we not getting a validation error here as was happening before?

Reply:
This PR disables the validator, no?

Reply:
Sorry my bad didn't read that right yep.